src/app/app.tsx

```diff
@@ -33,6 +33,9 @@ const router = createBrowserRouter(
             <Route path="/reset-password/:token" element={<ResetPassword />} />
             <Route path="/validate/success" element={<ValidateSuccess />} />
             <Route path="/validate/:token" element={<Validate />} />
+            <Route path="/public" element={<PublicLayout />}>
```

Comment:
Since this route is somewhat unrelated to the rest of the routes here I suggest moving it to the top (line 31).

Where has the route for /sign-in gone?